package.json

```diff
@@ -116,7 +116,8 @@
   "pnpm": {
     "overrides": {
       "jsdom": "24.0.0",
-      "@storybook/nextjs>postcss-loader": "8.1.1"
+      "@storybook/nextjs>postcss-loader": "8.1.1",
+      "@sentry/node>@opentelemetry/sdk-trace-base": "1.23.0"
```

Comment:
Otherwise getting type errors 
![telegram-cloud-photo-size-2-5258449901149346071-y](https://github.com/r34son/profile/assets/43549299/740f7097-e80a-49c8-8f6a-536fbccf2b25)
Because @opentelemetry/sdk-node uses @opentelemetry/sdk-trace-base: 1.23.0 I should pin exact version for @sentry/node dependency